### DISCUSSION TITLE
Authentication via Keycloak 20 fails with no error message

### DISCUSSION THREAD

**Discussion by aurorasmiles** [requarks/wiki#5805] [Help / Questions] [answered]
After upgrading to Keycloak 20 authentication with wikijs seems broken. 
After successfully authenticating in keycloak I get forwarded to a WikiJS error page, and I receive 500 Internal Server error.
There are no additional errors visible in the container logs (even when setting the log_level to silly), keycloak logs, browser console, reverse proxy logs. I have no idea how to start debugging this, since I don't even know what the issue is.

I'm using WikiJS 2.5.289 in Docker, PostgreSQL Database.

Edit: The same issue also seems to happen when using the generic OIDC adapter with Keycloak

**Comment by silicht** [comment, marked as answer]
Same issue here.

I created the pull request https://github.com/requarks/wiki/pull/5808 to fix it.

**Reply by aurorasmiles** [reply]
thank you very much!
